### ISSUE TITLE
[BUG] Use wire aliases for ACF2 compatibility

### ISSUE THREAD

**Issue by thegrb93** [Stooberton/ACF-3#38]
Wiremod has i/o aliasing so that renamed wire ports can still function with old dupes.

**Comment by thegrb93** [comment]
Example,

```
WireLib.AddInputAlias( "Zoom", "FOV" )
WireLib.AddOutputAlias( "XYZ", "HitPos" )
```

**Comment by thegrb93** [comment]
Ah wow they coded that really weird. Here's how ACF would use it

```
WireLib.AddInputAlias( "acf_ammo", "Active", "Load" )
```

**Comment by thegrb93** [comment]
Also, question. What is the Output <Vector> of acf_ammo for? I can't find what it is used for.

**Comment by TwistedTail** [comment]
Wasn't aware of this WireLib function, let me add it for the inputs that were just renamed. `Output [VECTOR]` is just a debug output Steve is using as a helper for the stuff he's doing with the entity hitboxes. It's gonna be removed once that work is done.

**Comment by thegrb93** [comment]
Sounds good, thanks!

**Comment by TwistedTail** [comment]
There's also `WireLib.AddOutputAlias`, gonna be using it once we readd the `AmmoCount` output to guns. Sadly, we won't be able to do the same with `Ready` since it was replaced with `Status [STRING]`.

**Comment by TwistedTail** [comment]
Fixed on a7bd507afe0f61df21daa96097e4c0e4179b55f3.
